draft-vkrasnov-h2-compression-dictionaries.md

```diff
@@ -223,14 +295,14 @@ optionally bundled with supporting implementations. Each dictionary should be
 Each static dictionary will be identified by an integer ID in the range {0..7}.
 
 If either endpoint supports the use of static dictionaries, it will indicate this
-by setting the SDVersion value of SETTINGS_COMPRESSION_SETTINGS to greater than
+by setting the SDVersion value of SETTINGS_COMPRESSION to greater than
 0. The number will indicate the highest version of the dictionaries known.
 
 The actual version used will be the lowest of the two values set by the endpoints.
 
-If the client and the server agreed on the use of static dictionaries, then
-both will initialize the first 8 dictionaries (IDs 0 through 7), with the contents
-of the static dictionaries. The Context ID for those dictionaries will be 0.
+If the client and the server agree on the use of static dictionaries, then both
+will initialize the first 8 dictionaries (IDs 0 through 7), with the contents
+of the static dictionaries. The static dictionaries belong to context 0.
```

Comment:
Why do we need context for the static dictionaries? Why can't they be used on streams that also use non-static dictionaries?

Reply:
To make things more simple. Context zero can be used as a dictionary to any context, and therefore you can build dictionaries from the static dictionaries for any other context. The context for a dictionary changes once you append a non-zero context stream to it.

Reply:
OK, I missed that part. Makes sense